ChangeLog.md

```diff
@@ -1,3 +1,8 @@
+# Changelog for tar-conduit
+
+## 0.2.5 - 2018-08-24
+ * Use forward slashes on Windows too, see [issue #21](https://github.com/snoyberg/tar-conduit/issues/21)
+
```

Comment:
The version `0.2.4` was never really released or uploaded to hackage, since I didn't get feedback for the lenient untaring. (I should have kept it in a separate branch, sorry, my mistake) @snoyberg, do you think we should combine that release with what I've written in #23? I still need to make some changes to for it to work on Windows, but I can get it done today.

Reply:
Yeah, I'm cool with that. Since everything seems broken right now, how about we get all of this merged into master and then clean it up there?